frontend/src/App.js

```diff
@@ -1,16 +1,46 @@
 import React from "react";
 import "./styles/App.css";
-import { BrowserRouter, Switch, Route } from "react-router-dom";
 
+import {BrowserRouter, Route, Switch} from "react-router-dom";
 // Components
 import Navigation from "./components/navigation";
 
+// import { BrowserRouter, Switch, Route } from "react-router-dom";
+// import { Link } from 'react-router-dom';
+// import QuantumSimulator from "./components/quantum";
+// import Header from "./components/header";
+// import Footer from "./components/footer";
+
+
+// // Components
+// import { makeStyles } from '@material-ui/core/styles';
+// import AppBar from '@material-ui/core/AppBar';
+// import Toolbar from '@material-ui/core/Toolbar';
+// import Typography from '@material-ui/core/Typography';
+// import Button from '@material-ui/core/Button';
+
 // Pages
 import SignupPage from "./pages/signup";
 import LoginPage from "./pages/login";
 import HomePage from "./pages/home";
+import TeacherTaskEditorPage from "./pages/teacherTaskEditor";
+import TeacherTasksPage from "./pages/teacherTasks";
+import TeacherTaskViewerPage from "./pages/teacherTaskViewer";
+
+const useStyles = makeStyles((theme) => ({
+	root: {
+	  flexGrow: 1,
+	},
+	signup: {
+	  marginRight: theme.spacing(3),
+	},
+	title: {
+	  flexGrow: 1,
+	},
+  }));
```

Comment:
It looks like `makeStyles` is being called whilst the import is commented out. Will this code work?

Reply:
kept this during resolving conflict.